pennylane/resource/__init__.py

```diff
@@ -15,7 +15,7 @@
 The ``resource`` module provides functionality to estimate the number of non-Clifford gates and
 logical qubits required to implement advanced quantum algorithms.
 """
-from .resource import Resources
+from .resource import Resources, count_resources

```

Comment:
Just want to open the conversation on _where_ this functionality is placed. Why do we put this in `pennylane/resource` rather than e.g., next to `specs` or the `Tracker`? I'm neutral towards this decision, but just want to capture the motivation.

Reply:
I think it can be in `pennylane/resource`. It will be used by both `specs` and `Tracker` and can be simply called from here.

Reply:
Keeping this in the resources module for now 👍🏼